src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArr = sourceString.trim().split(/;\s+|;/g);
+  stylesArr.pop();
+  const stylesObj = {};
```

Comment:
```suggestion
  const stylesArr = sourceString.trim().split(/;\s+|;/g);
  const stylesObj = {};
  
    stylesArr.pop();
```
я думаю буде достатньо `split(';')`